manager/assets/modext/workspace/lexicon/lexicon.grid.js

```diff
@@ -248,30 +247,46 @@ Ext.extend(MODx.grid.Lexicon,MODx.grid.Grid,{
     	this.loadWindow(btn, e, o);
     }
     ,reloadFromBase: function() {
-    	Ext.Ajax.timeout = 0;
-    	var topic = '/workspace/lexicon/reload/';
-        this.console = MODx.load({
-           xtype: 'modx-console'
-           ,register: 'mgr'
-           ,topic: topic
-        });
+        var tb = this.getTopToolbar(),
+        namespace = tb.getComponent('namespace').getValue(),
+        topicLexicon = tb.getComponent('topic').getValue(),
+        language = tb.getComponent('language').getValue(),
+        topic = '/workspace/lexicon/reload/';
```

Comment:
I'm just encouraging that any time javascript is being updated, we look to bring it to current coding conventions (namely ES6+). Also, I'd like to see us replacing abbreviated variable names to descriptive ones to improve readability. Here, and in any other place where the variable will not be re-defined, use `const` in the declaration.
```suggestion
        const toolbar = this.getTopToolbar(),
              namespace = toolbar.getComponent('namespace').getValue(),
              topic= toolbar.getComponent('topic').getValue(),
              language = toolbar.getComponent('language').getValue(),
              registryTopic = '/workspace/lexicon/reload/'
        ;
```